helm/sematic-server/templates/socketio_deployment.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.deployment.socket_io.dedicated }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-socketio
+  labels:
+    {{- include "sematic-server.labels" . | nindent 4 }}
+    app.kubernetes.io/component: socket_io
```

Comment:
Is it possible to stagger these two services' deployment?
When deploying a release that requires a DB migration, both of them would try to perform the migration, and I'm not sure what the implications would be.

Reply:
I think a better solution than that is to use a [pre-upgrade](https://helm.sh/docs/topics/charts_hooks/) hook in Helm to do the DB migrations.